src/cryptenroll/cryptenroll-custom.c

```diff
@@ -0,0 +1,133 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later
+ *
+ * Copyright © 2024 GNOME Foundation Inc
```

Comment:
We generally don't add copyright headers to source files anymore, as they are pretty much useless legally speaking, Bern convention and all that - is this a requirement from GNOME?

Reply:
Yes it's a requirement from GNOME

https://wiki.gnome.org/Foundation/OpenSourceGuidelines

Reply:
Thank you Sonny, that's great to hear!

Reply:
and I'm happy to list the Foundation among the contributors in NEWS for the next release

Reply:
We are well aware and very very happy about that project!